src/components/Tabs/Tabs.tsx

```diff
@@ -17,60 +17,69 @@ interface TabType {
 // prettier-ignore
 export interface TabsCustomProps {
     tabs : TabType[];
+    /** additional item to render inside the nav along with tab labels */
+    additionalNavItem?: React.ReactNode; 
```

Comment:
`additionalNavItem` => Does this mean only one element can be applied here?

What about something like this—

```
+-----------------------------------------+                          +-----------------------------------------+
|  tabs-labels-wrapper                    |                          |  additional-items-wrapper               |
________________________________________________________________________________________________________________
+--------------------------------------------------------------------------------------------------------------+
|  tabs content                                                                                                |
+--------------------------------------------------------------------------------------------------------------+

```
Also, maybe we need a better name than `additionalNavItem`, it could be a little more descriptive, what do you think?

Reply:
+1 on using a different name. Would `additionalContentWrapper` make sense?

Reply:
The idea behind naming it `item` and not `items` is that  in case there are multiple additional items, we expect it to be wrapped inside a single wrapper item. 

I'm looking for a better name as well, not able to think of something yet :')

Reply:
@ghostwriternr  `additionalNavContentWrapper`? since it's specifically for the nav.